R/colour-text.r

```diff
@@ -1,8 +1,9 @@
-
+# fix the check NOTE:Namespace in Imports field not imported from: ‘crayon’
```

Comment:
There should be no need to manually add `green`, `yellow` and `red` to the namespace - and in fact you could argue that crayon doesn't even need to be in the `Imports` field at all because it's not a run-time dependency (you only need it at build time).